modules/olm-mirroring-catalog.adoc

```diff
@@ -30,17 +31,13 @@ cluster on a restricted network, this registry can be a registry that the
 cluster has network access to, such as a mirror registry created during a
 restricted network cluster installation.
 
-You must also mirror the Red Hat-provided index image from Quay.io, or push your
+You must also mirror the Red Hat-provided index image, or push your
 own custom-built index image, to the target registry using the `oc image mirror`
 command. You can then use the mirrored index image to create a CatalogSource
 that allows Operator Lifecycle Manager (OLM) to load the mirrored catalog onto
 your {product-title} cluster.
 
-For this example, the procedure assumes the target registry is a previously
-created mirror registry that has access to both your restricted network and the
-internet. This example also shows mirroring the default `{catalog-name}`
-catalog, however it can be repeated for the other default catalogs, as well as
-custom catalogs.
+For this example, this procedure assumes the target registry is a previously created mirror registry that is accessible by both your cluster and a workstation with unrestricted network access. This example also shows mirroring the default `{catalog-name}` catalog, however it can be repeated for the other default catalogs, as well as custom catalogs.
```

Comment:
```suggestion
For this steps in this example, the target registry is an existing mirror registry that is accessible by both your cluster and a workstation with unrestricted network access. This example also shows mirroring the default `{catalog-name}` catalog, but the process is the same for all catalogs.
```

If you accept any of these changes, they're also valid in the pruning module. I also think that it is worth breaking down the catalog name format so that it's easier to see how to apply it for any catalog.